src/layouts/FellowLayout.tsx

```diff
@@ -54,12 +56,12 @@ export const FellowLayout = ({
           </Box>
           <Box
             w="300px"
-            display={{ base: "none", md: "block" }}
+            display={{ base: "none", md: "none" }}
```

Comment:
this would be the same as `display="none"`, so this element would be always hidden

Reply:
err... I wasn't sure if it was the right call to remove the index, but all the pages only have one <p> so it does not make sense to keep it.
Should it be removed at all? ....and not just CSS? it might be added in the future when we might have longer pages